src/skmultiflow/lazy/sam_knn.py

```diff
@@ -67,13 +67,15 @@ class SAMKNNClassifier(BaseSKMObject, ClassifierMixin):
     >>> from skmultiflow.data import FileStream
     >>> from skmultiflow.evaluation import EvaluatePrequential
     >>> # Setup the File Stream
-    >>> stream = FileStream("moving_squares.csv")
+    >>> stream = FileStream("https://raw.githubusercontent.com/scikit-multiflow/"
+    ...                     "master/moving_squares.csv")
```

Comment:
Same situation here.

Reply:
Fixed in e8b8f3fd3bb60bd1e6ad657e3fe45da11ac0a8d2